Bottom: 25e935c3767bc75f2481e9ceccaea9a6cc8bf884
Top:    4659458688860b29e082f2ff618892647dc74593
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 20:26:00 +0000

Refresh of add-raw-polygons-support-for-t

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 374fe5e..3f1c590 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -590,7 +590,6 @@ myVertex (GLdouble *vertex_data)
     printf ("Vertex recieved with unknown type\n");
 }
 
-
 void
 hidgl_fill_polygon (int n_coords, int *x, int *y)
 {
